scripts/referrer-user-count.ts

```diff
@@ -0,0 +1,74 @@
+import yargs from 'yargs'
+import { protocolFilters, supportedNetworkIds } from './consts'
+import { fetchReferralEvents, removeDuplicates } from './referrals'
+import { NetworkId, Protocol, protocols } from './types'
+import { writeFileSync } from 'fs'
+
+async function getArgs() {
+  const argv = await yargs
+    .env('')
+    .option('protocol', {
```

Comment:
add a type

Reply:
Added in f0073f9d16b11b770e9eb36dcad8008eb732797d.